make

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+###################################################
+#  Miking is licensed under the MIT license.
+#  Copyright (C) David Broman. See file LICENSE.txt
+#
+#  To make the build system platform independent,
+#  building is done using Dune and called from
+#  this bash script (on UNIX platforms) or
+#  using make.bat (on Windows).
+###################################################
+
+# Forces the script to exit on error
+set -e
+
+# Setup environment variable to find ipm folder
+export MCORE_STDLIB=`pwd`;
```

Comment:
I'm guessing this should be `MI_IPM`?

Reply:
Fixed

Reply:
Nice.  I'm not sure if there's a point in exporting this here though, come to think of it, since the script doesn't use this variable anywhere else.